db/versions/0091.yml

```diff
@@ -0,0 +1,391 @@
+version: 91
+description: migrate existing azure queue messages into new tables
+
+migrationScript: 0091-migration.sql
+downgradeScript: 0091-downgrade.sql
+
+methods:
+  # those methods and the azure messages table would be dropped in next release
+  azure_queue_get:
+    deprecated: true
+  azure_queue_count:
+    deprecated: true
+  azure_queue_delete:
+    deprecated: true
+  azure_queue_update:
+    deprecated: true
+  azure_queue_put_extra:
+    deprecated: true
+  azure_queue_delete_expired:
+    deprecated: true
```

Comment:
This feels like cheating a little, too -- any existing JS code still calling these functions will access the old table, potentially for example inserting a pending task, which new JS code will not see. And, this cheat really won't work if the old table is deleted! I suspect it's worthwhile re-implementing these functions in terms of the new queues. That might be difficult with the pending queues, though, since they are named after the taskQueueId. Maybe a not-quite-as-good solution is to just make all of these functions either return an empty set or fail, so that any old JS just fails and retries until the new JS starts running.

Reply:
Oops, my last comment ("In this context, "deprecated" means ...") should probably have been here.